output/cloud/output.go

```diff
@@ -452,82 +441,84 @@ func (out *Output) aggregateHTTPTrails(waitPeriod time.Duration) {
+					minReqDur, maxReqDur = reqDurations.SortGetNormalBounds(iqrRadius, iqrLowerCoef, iqrUpperCoef, true)
 				} else {
-					connDurations := make(durations, trailCount)
-					reqDurations := make(durations, trailCount)
-					for i, trail := range httpTrails {
-						connDurations[i] = trail.ConnDuration
-						reqDurations[i] = trail.Duration
-					}
+					minConnDur, maxConnDur = connDurations.SelectGetNormalBounds(iqrRadius, iqrLowerCoef, iqrUpperCoef)
+					minReqDur, maxReqDur = reqDurations.SelectGetNormalBounds(iqrRadius, iqrLowerCoef, iqrUpperCoef)
+				}
 
-					var minConnDur, maxConnDur, minReqDur, maxReqDur time.Duration
-					if trailCount < out.config.AggregationOutlierAlgoThreshold.Int64 {
-						// Since there are fewer samples, we'll use the interpolation-enabled and
-						// more precise sorting-based algorithm
-						minConnDur, maxConnDur = connDurations.SortGetNormalBounds(iqrRadius, iqrLowerCoef, iqrUpperCoef, true)
-						minReqDur, maxReqDur = reqDurations.SortGetNormalBounds(iqrRadius, iqrLowerCoef, iqrUpperCoef, true)
+				for _, trail := range httpTrails {
+					if trail.ConnDuration < minConnDur ||
+						trail.ConnDuration > maxConnDur ||
+						trail.Duration < minReqDur ||
+						trail.Duration > maxReqDur {
+						// Seems like an outlier, add it as a standalone metric
+						newSamples = append(newSamples, NewSampleFromTrail(trail))
 					} else {
-						minConnDur, maxConnDur = connDurations.SelectGetNormalBounds(iqrRadius, iqrLowerCoef, iqrUpperCoef)
-						minReqDur, maxReqDur = reqDurations.SelectGetNormalBounds(iqrRadius, iqrLowerCoef, iqrUpperCoef)
-					}
-
-					for _, trail := range httpTrails {
-						if trail.ConnDuration < minConnDur ||
-							trail.ConnDuration > maxConnDur ||
-							trail.Duration < minReqDur ||
-							trail.Duration > maxReqDur {
-							// Seems like an outlier, add it as a standalone metric
-							newSamples = append(newSamples, NewSampleFromTrail(trail))
-						} else {
-							// Aggregate the trail
-							aggrData.Add(trail)
-						}
+						// Aggregate the trail
+						aggrData.Add(trail)
 					}
 				}
+			}
 
-				aggrData.CalcAverages()
-
-				if aggrData.Count > 0 {
-					/*
-						out.logger.WithFields(logrus.Fields{
-							"http_samples": aggrData.Count,
-							"ratio":        fmt.Sprintf("%.2f", float64(aggrData.Count)/float64(trailCount)),
-							"t":            time.Since(start),
-						}).Debug("Aggregated HTTP metrics")
-					//*/
-					newSamples = append(newSamples, &Sample{
-						Type:   DataTypeAggregatedHTTPReqs,
-						Metric: "http_req_li_all",
-						Data:   aggrData,
-					})
-				}
+			aggrData.CalcAverages()
+
+			if aggrData.Count > 0 {
+				/*
+					out.logger.WithFields(logrus.Fields{
+						"http_samples": aggrData.Count,
+						"ratio":        fmt.Sprintf("%.2f", float64(aggrData.Count)/float64(trailCount)),
+						"t":            time.Since(start),
+					}).Debug("Aggregated HTTP metrics")
+				//*/
```

Comment:
I am not sure where this code is buried, maybe in a single commit that got reverted? Can't find it in the final diff, sorry :confused:

Reply:
I see it here: https://github.com/grafana/k6/blob/atlas/output/cloud/output.go#L510-L516

Reply:
This is a really old comment for debugging purposes. I think it has been around since the beginning of the cloud output aggregation.

Reply:
Yeah, it is in `master` too: https://github.com/grafana/k6/blob/master/output/cloud/output.go#L518-L524
I guess it showed on your diff because the indentation changed and not on mine because I've disabled that in github diffs